indico/web/forms/fields/simple.py

```diff
@@ -147,5 +147,15 @@ def _value(self):
         return self.text_value
 
 
+class IndicoEmailRecipientsField(Field):
+    widget = JinjaWidget('forms/email_recipients_widget.html', single_kwargs=True)
+
+    def process_data(self, data):
+        self.data = data
+        self.text_value = ', '.join(sorted(x.email for x in data))
```

Comment:
I would already pass a list of emails to the field. Otherwise it can't be used  in places where you don't have an object with an `email` attribute.